src/components/movieForm/index.tsx

```diff
@@ -112,33 +184,57 @@ const MovieForm: React.FC<IMovieFormProps> = ({
             data-testid='dropdown'
             onOptionSelect={handleOptionSelect}
             multiselect={true}
+            defaultSelectedOptions={initialMovieInfo?.genres ?? []}
             selectedOptions={selectedOptions}
-            value={value}
             placeholder='Select genres'
+            name='genres'
+            defaultValue={value}
```

Comment:
You need to add validation for this field also: 
<img width="526" alt="image" src="https://user-images.githubusercontent.com/81073/235857198-c8c9f5af-29a6-420b-92b7-b6c80d5e5dd8.png">